guides/common/modules/proc_creating-a-webhook.adoc

```diff
@@ -3,10 +3,18 @@
 [id="creating-a-webhook_{context}"]
 = Creating a webhook
 
+[role="_abstract"]
 You can customize events, payloads, HTTP authentication, content type, and headers through the {ProjectWebUI}.
 
 Use the following procedure to create a webhook in the {ProjectWebUI}.
 
+[NOTE]
+====
+ifndef::foreman-deb,orcharhino[]
```

Comment:
Please move the ifndef outsite the note. I wonder if this evalutes to false, asciidoctor creates an empty NOTE admonition?